Makefile

```diff
@@ -126,7 +126,7 @@ test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -v ./apis/... ./controllers/... -coverprofile cover.out
 
 .PHONY: integ-test
-integ-test: kustomize cert-manager ## Run integration tests from directory M_INTEG_DIR or set M_INTEG_DIR=all to run all the integration tests.
+integ-test: kustomize cert-manager ./bin/helm ## Run integration tests from directory M_INTEG_DIR or set M_INTEG_DIR=all to run all the integration tests.
```

Comment:
Just do helm here, which would lead to ``helm:`` block. When calling Helm, we should probably call it using $(HELM) instead of directly with /bin/helm. Or even check if the system has it with ``which`` command. We do that for some of the tools. Also, all these parts should be prefixed with ``.PHONY`` to avoid looking a file instead of Makefile target. 

Lets not mix up Makefile targets with filenames.

Reply:
I quite like having the file target actually, as I run into frequent issues with tool installations where (even where the tool isn't present) they often don't run the installation step as (for some reason) make thinks the tool is already installed.

Am I using make wrong perhaps?

Reply:
I've never seen make not run the target. I think there's probably been some other case that causes it.

If the match is a file, make will not execute that block in your Makefile. It will also always call the file first before the Makefile target.